src/main/java/com/spotinst/sdkjava/example/elastigroup/aws/ElastigroupUsageExample.java

```diff
@@ -715,33 +716,23 @@ private static void detachLoadBalancer(SpotinstElastigroupClient client, String
         }
     }
 
-    private static void enterGroupStandby(SpotinstElastigroupClient client, String elastigroupId) {
-
-        // Build standby elastigroup request
-        ElastigroupStandbyRequest.Builder elastigroupStandbyRequestBuilder = ElastigroupStandbyRequest.Builder.get();
-        ElastigroupStandbyRequest postRequest =
-                elastigroupStandbyRequestBuilder.setElastigroupId(elastigroupId).build();
+    private static void enterInstanceStandby(SpotinstElastigroupClient client, String instanceId) {
```

Comment:
can we remove BL and API class for this "ElastigroupStandbyRequest", hence this is not used with latest changes